themes/default/content/blog/deploy-azure-static-website/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Introducing the Azure Static Website Component
+date: 2022-10-11
+meta_desc: Learn how to quickly deploy your static site to Azure using the Azure Static Website Component
+meta_image: "meta_img.png"
+authors:
+    - sean-holung
+tags:
+    - azure
+    - configuration
+    - components
+    - static-websites
+---
+
+Deploying a static website often involves provisioning a number of pieces of infrastructure and stitching those pieces together in a way to make the site accessible to your users. A static website typically consists of HTML, CSS, and JavaScript files along with any other static assets the site might need to render such as PNG and SVG files for example. These files are then uploaded to a storage bucket where they can be served from. This post will demonstrate how to easily deploy your static website to Azure and make it available for public access.
+
+## Azure Static Website Component
+
+Pulumi components enable us to create re-usable abstractions that can perform complex tasks all behind a simple interface the user can programmatically interact with. The [Azure Static Website component]({{< relref "/registry/packages/azure-static-website/" >}}) looks to take all of the complexity of getting a static website deployed to Azure and simplify it down to just consuming a simple component and passing it a few input arguments. This component relieves the complexities of this process by allowing you to simply tell the component where your files are located and it will take care of all the rest.
+
+**This component does the following:**
+
+* Provisions an Azure storage account
+* Uploads the website contents located in the specified directory
+* Optionally can configure a CDN, custom domain, and certs.
+
+## Using the component to deploy a static website
+
+Let’s walk through a simple example of using this component to deploy a static website. Today, we are going to provision a storage account, upload the website contents, and wire it up to an Azure CDN.
+
+In order to follow along with this example you will need to have the following:
+
+* Pulumi CLI installed on your machine
```

Comment:
I think adding some links how to install Pulumi and set up Azure would be helpful.